content/press.adoc

```diff
@@ -94,3 +94,6 @@ volunteers so availability and time may be limited.
 * link:https://github.com/batmat[Baptiste Mathus] (_french, english_) - Contributor / Meetup Organizer - `bmathus[at]batmat.net`
 * link:https://github.com/orrc[Christopher Orr] (_english, german_) - Contributor / Speaker - `chris[at]orr.me.uk`
 
+=== CIS press contacts
```

Comment:
I'm not an english native speaker, but neither did I know that acronym. But that doesn't mean much, IANAJ :)

/me learns about it https://en.wikipedia.org/wiki/Commonwealth_of_Independent_States